greeting.js

```diff
@@ -1,4 +1,5 @@
 var greet = function(name){
+	"use strict";
```

Comment:
I know why you put this here, but normally you wouldn't do it like this. No worries, will talk more about it later...